jobs/JGFS_ATMOS_GEMPAK_PGRB2_SPEC

```diff
@@ -27,6 +14,9 @@ export finc=3
 export model=gfs
 export EXT=""
 
+# For half-degree P Grib files
+#export DO_HD_PGRB=YES
+
```

Comment:
??
Should this be in a `config.gempak_spec` file?

Reply:
Probably. @WalterKolczynski-NOAA is this something you could pull out of this script and into a config within your gempak work?